### ISSUE TITLE
New immersive mode

### ISSUE THREAD

**Issue by ebraminio** [SufficientlySecure/document-viewer#131]
Just as Todo, Android 5.0 got new feature which makes system status bar transcluent, probably it can be levereged on fullscreen mode of the app.

**Comment by KrasnayaPloshchad** [comment]
Well, let me correct, the immersive mode is introduced in Android 4.4.

**Comment by ebraminio** [comment]
Yep, I tried to use that on the app [while ago](https://github.com/SufficientlySecure/document-viewer/pull/39) ☺

**Comment by ebraminio** [comment]
Hmm right, I tried to use that on the app [while ago](https://github.com/SufficientlySecure/document-viewer/pull/39) actually but the bug title and description are misleading. What I meant was the new feature, transcluent statusbar http://blog.raffaeu.com/archive/2015/04/11/android-and-the-transparent-status-bar.aspx which we should try to implement.

**Comment by ericwa** [comment]
Ah, so this feature would be a transparent statusbar for when you're not in fullscreen mode? Like Google PDF Viewer?
![screen5out](https://cloud.githubusercontent.com/assets/239161/11165014/5663a290-8abf-11e5-8966-402584ad523e.png)

**Comment by ebraminio** [comment]
Yes :)
